packages/manager-components/src/components/guides-header/guides-header.component.tsx

```diff
@@ -60,7 +60,9 @@ export function GuidesHeader({
         <GuidesHeaderItem
           key={guide}
           guide={guides[guide]}
-          href={`${guides[guide].url[ovhSubsidiary]}`}
+          href={
+            guides[guide].url[ovhSubsidiary] || guides[guide].url["DEFAULT"]
```

Comment:
```suggestion
            guides[guide].url[ovhSubsidiary || 'DEFAULT']
```

Reply:
What you proposed doesn't fix my problem. If we suppose that the subsidiary exists (like FR, for example), but there is no FR entry in my guidesUrl object:

```
{
  GB: 'url-gb',
  IE: 'url-ie',
  DEFAULT: 'default-url',
  US: 'url-us',
}
```

In this case, I want to use the default URL. With your proposal, `guides[guide].url[ovhSubsidiary || 'DEFAULT']`, the output would be `undefined`.